### PR TITLE
Hotfix api_wraper.py responed server error msg for the first try| Optimized the berry_inventory function called times

### PR DESCRIPTION
**Pull request by xc-mezcal**
Short Description: 

*************************************This is wrong guess but a quite close fix **********************
Some time, the `result = self._api.call()` is getting response but not filled with full response msg. Then the check begins,

**************************************The True may be ************************************
We believe that server now detects massive requests, so we put a force sleep after an API call. 

So the speed of making api request is reduced to hot fix this problem
*********************************************************************************************

This makes the every first response check failed (because it's still point to empty) and second response succeed. 

Fixes:
Add a wait time to "wait" the result to be "fully filled" solve this problem for me.

### CODE REVIEW

File: pokemongo_bot/cell_workers/pokemon_catch_worker.py [hunk 1]
```diff
@@ -78,6 +78,9 @@ def work(self):
                             return False
 
                         items_stock = self.bot.current_inventory()
+                        berry_id = 701 # @ TODO: use better berries if possible
```
Comment (TheSavior) [inline pokemongo_bot/cell_workers/pokemon_catch_worker.py]: This change seems irrelevant to this PR.

Comment (xc-mezcal) [inline pokemongo_bot/cell_workers/pokemon_catch_worker.py]: I don't know how git hub PR works.

But it automatically added my second PR to this one. lol
